### PR TITLE
Added: Walkthrough: Create individual Options with Checkboxes

### PR DESCRIPTION
**Pull request by SFC-Sarge**
@madskristensen;

I added: 
> Walkthrough: Create individual Options with Checkboxes

 to this, so other developers will not have to ask how in the future.

### CODE REVIEW

**Comment by SFC-Sarge** [comment]
@madskristensen; Again it modified additional files, and I do not remember how I stopped it from changing those files. I opened VS 2020 17.1 and then opened the doc project again and it shows the following without doing anything but opening the GitHub tool window:
image

The only files that I added or changed were these:

image

File: .vs/ProjectSettings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (madskristensen) [inline .vs/ProjectSettings.json:1]: did you mean to commit the files in the .vs folder?

**Comment by madskristensen** [comment]
Can you update the .gitignore file to make sure it ignores the .vs folder?

File: .vs/ProjectSettings.json (hunk 1 above)
Comment (SFC-Sarge) [reply .vs/ProjectSettings.json:1]: This is my Visual Studio Explore show all files. Even after changing the .gitignore file.

image


This is the contents of the .gitignore file:

_site
.jekyll-cache
.vs
.vs\VSWorkspaceState.json
.vs\slnx.sqlite
.vs\ProjectSettings.json
.vs\docs
.vs\docs\v17
.vs\docs\v17\.suo

image


I cllosed Visual Studio, Opened Visual Studio and yet as soon as I changed/updated the `settings-and-options.md they show up again as changed.

image

**Comment by madskristensen** [comment]
Looks like the .vs folder is still present in your fork: https://github.com/VsixCommunity/docs/tree/fb745021ff5ee38c5b9fc111a37c0ed1bd4d19fa/docs/.vs

**Comment by madskristensen** [comment]
Just delete the .vs folders too and it should be good to go

**Pull request merged**